Library/brew.sh

```diff
@@ -201,6 +169,64 @@ EOS
   esac
 fi
 
+if [[ -z "$HOMEBREW_DEVELOPER" ]]
+then
+  unset HOMEBREW_RUBY_PATH
+fi
+
+if [[ -z "$HOMEBREW_RUBY_PATH" && "$HOMEBREW_COMMAND" != "install-vendor" ]]
+then
+  HOMEBREW_VENDOR_RUBY_PATH="$HOMEBREW_LIBRARY/Vendor/Ruby/Versions/Current/bin/ruby"
+  if [[ -x "$HOMEBREW_VENDOR_RUBY_PATH" ]]
+  then
+    HOMEBREW_RUBY_PATH="$HOMEBREW_VENDOR_RUBY_PATH"
+  else
+    if [[ -n "$HOMEBREW_OSX" ]]
+    then
+      HOMEBREW_RUBY_PATH="/System/Library/Frameworks/Ruby.framework/Versions/Current/usr/bin/ruby"
+
+      if [[ -d "/System/Library/Frameworks/Ruby.framework/Versions/1.8" ]]
```

Comment:
This is for OS X to speed up performance by avoiding unnecessary subprocess. Noted that most of users will just continue to use Ruby 2.0 offered by Apple.